perf/benchmarkapps/BenchmarkServer/grpc-aspnetcore.json

```diff
@@ -0,0 +1,27 @@
+{
+  "Default": {
+    "Client": "Grpc",
+    "ClientProperties": {
+      "UseTls": "false"
+    },
+
+    "Source": {
+      "Repository": "https://github.com/aspnet/grpc-dotnet.git",
```

Comment:
grpc/grpc-dotnet? Also maybe the BranchOrCommit should also be changed to master upon merge?